argus/backends/local/local_backend.py

```diff
@@ -0,0 +1,46 @@
+from argus.backends import base as base_backend
+from argus.backends import windows as windows_backend
+from argus.client import windows
+from argus import config as argus_config
+from argus import log as argus_log
+from argus import util
+
+CONFIG = argus_config.CONFIG
+
+
+class LocalBackend(windows_backend.WindowsBackendMixin,
+                   base_backend.BaseBackend,
```

Comment:
Slight(OCD) suggestion, wouldn't this look more readable?
`class LocalBackend(windows_backend.WindowsBackendMixin,
                                windows_backend.BaseMetadataProviderMixin,
                                base_backend.BaseBackend):`